Cargo.lock

```diff
@@ -336,7 +336,6 @@ dependencies = [
  "slog-term",
  "store",
  "tokio 0.2.21",
- "toml",
```

Comment:
I'm not sure what happened here..

Reply:
I've seen this on a few other branches too, nothing to worry about